src/Bicep.Core/Registry/DefaultArtifactRegistryProvider.cs

```diff
@@ -29,7 +29,7 @@ public DefaultModuleRegistryProvider(IServiceProvider serviceProvider, IFileReso
             this.serviceProvider = serviceProvider;
         }
 
-        // The templateUri affects how module aliases are resolved (depending on whether the bicepconfig.json is located for the given template)
+        // NOTE(stephwe): The templateUri affects how module aliases are resolved (depending on whether the bicepconfig.json is located for the given template)
```

Comment:
[nit]: I think the `note(<alias>):` convention is really only used for internal repositories, where you can use an internal system to map `<alias>` to a specific person. I only found six instances of this in the Bicep repo, and they're all recent; the standard has been to just leave a comment and let users figure out the source via `git blame` if needed.

Reply:
The problem with using blame is that when you do a copy paste the information becomes opaque. I think we could use github aliases and that should work well enough to map to a specific person no?